src/components/cesium_map/cesium_UI.js

```diff
@@ -216,7 +227,7 @@ class CesiumMap extends React.Component {
       if (diffDistance > 50 + 4000 * viewer.currentView.height / 15000000) {
```

Comment:
Numbers like this are the kind of things @datadavev mentioned should be extracted out to constants.  I realize you didn't change these as part of this PR, but it would be good to follow up and make a subsequent PR to clean these (and any others) up.

Reply:
OK, I will make a subsequent PR to clean it up.